astrapy/db.py

```diff
@@ -848,6 +848,19 @@ def delete_many(self, filter: Dict[str, Any]) -> API_RESPONSE:
 
         return response
 
+    def truncate(self) -> API_RESPONSE:
```

Comment:
Maybe call it `clear` ? (same as cassio)

Reply:
We could also make `{}` the default filter of `delete_many` so `collection.delete_many()` would delete all.

Reply:
Agree on clear - I suggest however to keep truncate as an alias (it exists already and is theoretically used around)

As for the default in deleta_many, I am a bit afraid of the potential risk of wiping out inadvertently all your stuff, no?

Reply:
OK. Let's go for clear then.